src/types/products.ts

```diff
@@ -38,7 +38,7 @@ type LearnProductName = Exclude<
  * interface almost the same as `ProductData`, just with a limited set of
  * options for `slug`.
  */
-interface LearnProductData extends ProductData, LearnProduct {
+interface LearnProductData extends ProductData {
```

Comment:
[See this note](https://github.com/hashicorp/dev-portal/pull/278/files#r845582069)